layout/js/svg.js

```diff
@@ -20,19 +20,22 @@ var vizlab = vizlab || {};
       return point;
     }
     
+    var svgId = $(svg).attr("id");
+    var tooltipGroupId = svgId + '-tooltip-group';
+    
     var TOOLTIP_HTML = 
       '<defs>' + 
-      '<clipPath id="tipClip-test">' +
+      '<clipPath id="' + svgId + '-tipClip">' +
       '<rect x="-6" y="-11.5" height="11" width="12"/>' +
       '</clipPath>' + 
       '</defs>' + 
-      '<rect id="tooltip-box-test" height="24" class="tooltip-box"/>' +
-      '<path id="tooltip-point-test" d="M-6,-12 l6,10 l6,-10" class="tooltip-box" clip-path="url(#tipClip-test)"/>' +
-      '<text id="tooltip-text-test" dy="-1.1em" text-anchor="middle" class="svg-text"> </text>';
+      '<rect height="24" class="tooltip-box hidden"/>' +
+      '<path d="M-6,-12 l6,10 l6,-10" class="tooltip-point hidden" clipPath="url(#'+ svgId + '-tipClip"/>' +
```

Comment:
I may be wrong, it looks like clip-path and clipPath may both work? One being the css syntax and the other being svg? 😕